cypress/integration/advanced-functionalities/server/Advanced_functionalities.spec.js

```diff
@@ -0,0 +1,106 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const newEmail = Cypress.env('user').newEmail
+const newPassword = Cypress.env('user').newPassword
+
+describe('Test server advanced functionalities ', () => {
+  beforeEach(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ baseURL })
+      cy.get('[data-test-id="input-email"]').type(newEmail)
+      cy.get('[data-test-id="input-password"]').type(newPassword)
+      cy.get('[data-test-id="button-login-and-remember"]').click()
+    }
+  })
+
+  context('Testing your profile tab', () => {
+    it('should be able to change the name', () => {
+      cy.visit(baseURL + '/auth/login')
+      cy.get('[data-test-id="input-name"]').type(' edited')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="input-name"]').should('have.value', 'Permissions account edited')
+    })
+
+    it('should be able to change the handle', () => {
+      cy.get('[data-test-id="input-handle"]').type('_edited')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="input-handle"]').should('have.value', 'permissions_account_edited')
+    })
+
+    it('should be able to change the preferred language', () => {
+      cy.get('[data-test-id="select-language"]').select('German (German)')
+      cy.get('[data-test-id="button-submit"]').click()
+      // We check if the success toast appears
+      cy.get('.border-primary')
+      cy.get('[data-test-id="select-language"]').should('exist', 'German (German)')
+    })
+  })
+
+  // // Currently there is a bug in client authorization, so excluding the revoke access check
+  // context('Testing authorized clients tab', () => {
+  //   it('should be able to revoke access', () => {
+  //     // We first visit webapp compose in order to make sure that we will have data under authorized clients
+  //     cy.visit(baseURL + '/compose/namespaces')
+  //     // We wait 2s in order the page to be loaded
+  //     cy.wait(2000)
+  //     cy.visit(baseURL + '/auth')
+  //     cy.get('[data-test-id="link-tab-authorized-clients"]').click()
+
+  //     cy.get('[data-test-id="button-revoke-access"]').should('exist').click()
+  //     // We check if the success toast appears
+  //     cy.get('.border-primary')
+  //     cy.get('[data-test-id="button-revoke-access"]').should('not.exist')
+  //   })
+  // })
+
+  context('Testing username link', () => {
+    it('should be able to click on user name and be redirected to your profile tab', () => {
+      cy.get('[data-test-id="link-redirect-to-profile"]').click()
+      cy.url().should('exist', baseURL + '/auth/')
+    })
+  })
+
+  context('Testing log in button', () => {
+    it('should be able to log out and press on log in from the modal', () => {
```

Comment:
wasn't this tested already? You said so in your ticket

Reply:
Removing this test.